test/board.test.ts

```diff
@@ -1,28 +1,27 @@
-import { Board } from '../src/Board'
-import { Team, Color } from '../src/constants/Constants'
+import { Board } from '../src/Board';
+import { Team, Color } from '../src/constants/Constants';
 
 import 'mocha';
 import { expect } from 'chai'
 
 describe("Filename: board.test.ts:\n\nThe board", () => {
-    it("should have expected number of colors with expected distribution", () => {
-        let board = new Board(Team.blue);
+  it("should have expected number of colors with expected distribution", () => {
+    let board = new Board(Team.blue);
+    
+    let num_red = board.colors.filter(n => n === Color.red).length;
+    let num_blue = board.colors.filter(n => n === Color.blue).length;
+    let num_neutral = board.colors.filter(n => n === Color.neutral).length;
+    let num_assassins = board.colors.filter(n => n === Color.assassin).length;
+    
+    expect(board.colors.length).to.equal(25);
+    expect(num_red).to.be.within(8, 9);
```

Comment:
Since the starting team is blue, wouldn't blue always have 9 cards, and red always 9? Testing this myself showed this to be a correct assumption, so I'd suggest changing this test to specify those and documenting why that is. Alternatively, you could make tests that verify both blue and red having 9 cards if they are the current team.